src/libraries/System.Text.RegularExpressions/gen/Stubs.cs

```diff
@@ -90,7 +90,9 @@ namespace System.Text.RegularExpressions
 {
     internal sealed class RegexReplacement
     {
+#pragma warning disable IDE0060
```

Comment:
You can just put this at the top of the file and leave off the restore.  The whole file is here to provide polyfills.

Reply:
ok